ui/app/components/wizard/features-selection.js

```diff
@@ -16,7 +18,25 @@ export default Component.extend({
       feature.show = false;
     }
   },
-
+  estimatedTime: computed('selectedFeatures', function() {
+    let time = 0;
+    for (let feature of Object.keys(FEATURE_MACHINE_TIME)) {
+      if (this.selectedFeatures.includes(feature)) {
+        time += FEATURE_MACHINE_TIME[feature];
+      }
+    }
+    return time;
+  }),
+  selectProgress: computed('selectedFeatures', function() {
+    let bar = [];
+    this.selectedFeatures.forEach(feature => {
```

Comment:
Could do `let bar = this.selectedFeatures.map...` here so then you wouldn't need the push.